english/interact/run-into-wall/run-into-wall.md

```diff
@@ -0,0 +1,6 @@
+You decide that you know marshmallows are soft.  How could they be solid walls? What would happen if you just tried to run through them?  It probably wouldn't hurt to run into a wall of marshmallows right?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.